velox/exec/HashTable.cpp

```diff
@@ -229,7 +229,8 @@ class ProbeState {
       return group_;
     }
     const auto kEmptyGroup = BaseHashTable::TagVector::broadcast(kEmptyTag);
-    for (;;) {
+    for (int64_t numProbedBuckets = 0; numProbedBuckets < table.numBuckets();
+         ++numProbedBuckets) {
```

Comment:
@xiaoxmeng When searching in a bucket, continue will increase numProbedBuckets. Is this behavior expected?

```
        loadNextHit<Operation::kProbe>(
            table, -static_cast<int32_t>(sizeof(normalized_key_t)));
        if (RowContainer::normalizedKey(group_) == keys[row_]) {
          table.incrementHits();
          return group_;
        }
        continue;
```